doc/mkdocs/docs/api/basic_json/value_t.md

```diff
@@ -22,12 +22,43 @@ functions [`is_null`](is_null.md), [`is_object`](is_object.md), [`is_array`](is_
 [`is_discarded`](is_discarded.md), [`is_binary`](is_binary.md), [`is_primitive`](is_primitive.md), and
 [`is_structured`](is_structured.md) rely on it.
 
+## Ordering
```

Comment:
Can you move this to the Notes section? We have standardized sections throughout the documentation (see check_structure.py).